package.json

```diff
@@ -102,29 +105,25 @@
     "webpack-postcss-tools": "^1.1.1"
   },
   "dependencies": {
-    "@kadira/storybook": "^1.36.0",
-    "autoprefixer": "^6.3.3",
-    "babel-plugin-transform-builtin-extend": "^1.1.0",
     "classnames": "^2.2.5",
     "create-react-class": "^15.6.0",
     "dateformat": "^1.0.12",
     "deep-equal": "^1.0.1",
     "fuzzy": "^0.1.1",
+    "gotrue-js": "^0.9.3",
```

Comment:
I moved `gotrue-js` into the regular dependencies since that is consistent with what we have right now.